source/access/mfa_login.rst

```diff
@@ -3,23 +3,167 @@
+
+Connecting without an SSH agent
+-------------------------------
+
+Using SSH clients (such as PuTTY or terminal) is easier to setup *without*
+an :ref:`SSH agent <SSH agent>`; however, on a long run, it involves repetitive authentication
+which is not convenient.
+Hence, we do not recommend this approach.
+But, if you opt for this approach, here are the steps to follow:
+
+- You login via terminal (or PuTTY or MobaXterm) to one of the
+  :ref:`Tier-2 login nodes <tier2_login_nodes>`.
+  You are prompted with a login link:
+
+  .. _firewall_link_mfa:
+  .. figure:: mfa_login/firewall_link_mfa.PNG
+     :alt: firewall_link_mfa
+
+  Copy-paste the provided link in a browser and follow it.
+  Note that when using PuTTY or MobaXterm, simply highlighting the link with your
+  mouse will copy the URL to your clipboard.
+  Avoid using 'CTRL-C', or it will send a ``SIGINT`` signal interrupting
+  your process instead of performing a copy operation.
+
+- From the drop-down menu, choose the institute you are affiliated with.
+  Below, we show an example of a KU Leuven user, but one has to pick the
+  institute he/she is affiliated with.
+
+  .. figure:: mfa_login/vsc_firewall_institute.PNG
+     :alt: Choose your institute
+
+- You will be forwarded to the Identity Provider (IdP) of your institute,
+  and you need to login in a usual way using your registered credentials.
+  For KU Leuven users, the page looks like the following:
+
+  .. _idp_page:
+  .. figure:: mfa_login/idp_page.PNG
+     :alt: idp_page
+
+- If you are already connected to the internal network, then you will be only asked to
+  identify yourself with the MFA authenticator app on your personal phone:
+
+  .. _reauthenticate_phone:
+  .. figure:: mfa_login/reauthenticate_phone.PNG
+     :alt: reauthenticate_phone
+
+  This step may not be necessary when connecting from a white-listed IP address,
+  like the internal networks of the Flemish universities, using a static on-site
+  IP as well as the institutional VPN.
+  For example, if you have already logged upfront into your institution's network
+  then you might not be required to log in again depending on your browser
+  session settings (e.g., accepted cookies).
+
+- Once you are successfully authenticated, you end up on a page telling you that your VSC 
+  identity is confirmed.
+  If you have already performed the previous login in that browser session, you will 
+  immediately end up on this page:
+
+  .. _firewall_confirmed:
+  .. figure:: mfa_login/firewall_confirmed.PNG
+     :alt: firewall_confirmed
+
+- Go back to your browser, SSH client or terminal.
+  You should now be connected to a login node on the cluster or to the OnDemand web portal,
+  respectively.
+  E.g. for a PuTTY session, a successful login is rewarded with
+  a welcome message:
+
+   .. _login_node:
+   .. figure:: mfa_login/login_node.PNG
+      :alt: login_node
+
+That's it! You can continue doing your HPC work as usual.
+
+.. note::
+
```

Comment:
do we need to specify this note in the documentation? I would pass this more on a need-to-kown basis. But maybe that's me being to prudent?

Reply:
removed in ffe64e5